components/list/list-item-mixin.js

```diff
@@ -695,7 +696,7 @@ export const ListItemMixin = superclass => class extends composeMixins(
 					class="d2l-list-item-actions-container">
 					<slot name="actions" class="d2l-list-item-actions">${actions}</slot>
 				</div>
-				${this._showAddButton && !this._hasNestedListAddButton ? html`
+				${this._showAddButton && (!this._hasNestedListAddButton || (this.expandable && !this.expanded)) ? html`
```

Comment:
We want to show the bottom add button only when it's not expandable, or if it is expandable then it must not be expanded.